config/initializers/test.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+I18n.exception_handler = Proc.new do |exception, *_|
+  raise exception.to_exception
```

Comment:
Unfortunately this also raises exceptions in production. It's not limited to test. It would need to go into the base spec helper or a spec support file.

Reply:
Ok - I've moved this into base spec helper on the latest commit :+1: 

PS: what if we add a rescue statement to notify Bugsnag? This way we could catch missing translations in pages which are not covered by specs, but visited by users in production... The downside could be some false positives as well. Not sure it would be a good idea.

Reply:
> what if we add a rescue statement to notify Bugsnag?

That's a brilliant idea! Well, except that we don't want to be warned about other locales than en.yml and pretty much all production instances use their own locales. :disappointed: 

So maybe one day, when we assume that all production locales are 100% translated. In the meantime, let's not add that noise to our Bugsnag stream.